components/molecules/NivoScatterChart/nivo-scatter-chart.tsx

```diff
@@ -27,21 +35,42 @@ interface ScatterPlotProps {
   }[];
   isMobile?: boolean;
   repositories?: number[];
+  metadata: ScatterChartMetadata;
+  handleSetPrFilter: (state: "open" | "closed" | "all") => void;
```

Comment:
Could you create a new type 
`type StateOfPRs="open" | "closed" | "all"` 
I saw that you have to use this type 2 times

Reply:
Yeah that's valid. Done